### PR TITLE
include psycopg3 benchmark

### PR DESCRIPTION
**Pull request by farahats9**
[asyncpg issue #924](https://github.com/MagicStack/asyncpg/issues/924)

### CODE REVIEW

File: _python/pgbench_python.py [hunk 1]
```diff
@@ -45,14 +45,22 @@ def psycopg_connect(args):
                             port=args.pgport)
     return conn
 
+def psycopg2_connect(args):
```
Comment (elprans) [inline _python/pgbench_python.py:48]: Wrong module used above on line 44, should be `psycopg.connect`

Comment (farahats9) [reply _python/pgbench_python.py:48]: thanks for that.

File: _python/pgbench_python.py [hunk 2]
```diff
@@ -128,15 +159,33 @@ async def asyncpg_connect(args):
     return conn
 
 
+async def async_psycopg_connect(args):
+    conn = await psycopg.AsyncConnection.connect(user=args.pguser, host=args.pghost,
+                                 port=args.pgport)
+    return conn
```
Comment (elprans) [inline _python/pgbench_python.py]:
```suggestion
async def async_psycopg_connect(args):
    conn = await psycopg.AsyncConnection.connect(
        user=args.pguser, host=args.pghost, port=args.pgport)
    return conn
```

File: _python/pgbench_python.py [hunk 3]
```diff
@@ -369,7 +433,7 @@ def die(msg):
         help='PostgreSQL server user')
     parser.add_argument(
         'driver', help='driver implementation to use',
-        choices=['aiopg', 'aiopg-tuples', 'asyncpg', 'psycopg', 'postgresql'])
+        choices=['aiopg', 'aiopg-tuples', 'asyncpg', 'psycopg2', 'psycopg3', 'psycopg3[async]', 'postgresql'])
```
Comment (elprans) [inline _python/pgbench_python.py]:
```suggestion
        choices=[
            'aiopg',
            'aiopg-tuples',
            'asyncpg',
            'psycopg2',
            'psycopg3',
            'psycopg3-async',
            'postgresql'
        ],
```

File: _python/pgbench_python.py [hunk 4]
```diff
@@ -425,11 +489,21 @@ def die(msg):
             asyncpg_connect, asyncpg_execute, asyncpg_copy, asyncpg_executemany
         is_async = True
         arg_format = 'native'
-    elif args.driver == 'psycopg':
-        connector, executor, copy_executor = \
-            psycopg_connect, psycopg_execute, psycopg_copy
+    elif args.driver == 'psycopg2':
+        connector, executor, copy_executor, batch_executor = \
+            psycopg2_connect, psycopg2_execute, psycopg2_copy, psycopg2_executemany
```
Comment (elprans) [inline _python/pgbench_python.py]:
```suggestion
        connector, executor, copy_executor, batch_executor = (
            psycopg2_connect, psycopg2_execute, 
            psycopg2_copy, psycopg2_executemany,
        )
```

File: _python/pgbench_python.py [hunk 5]
```diff
@@ -425,11 +489,21 @@ def die(msg):
...
         is_async = False
         arg_format = 'python'
+    elif args.driver == 'psycopg3':
+        connector, executor, copy_executor, batch_executor = \
+            psycopg_connect, psycopg_execute, psycopg_copy, psycopg_executemany
+        is_async = False
+        arg_format = 'python'
+    elif args.driver == 'psycopg3[async]':
+        connector, executor, copy_executor, batch_executor = \
+            async_psycopg_connect, async_psycopg_execute, async_psycopg_copy, async_psycopg_executemany
```
Comment (elprans) [inline _python/pgbench_python.py]:
```suggestion
        connector, executor, copy_executor, batch_executor = (
            async_psycopg_connect, async_psycopg_execute, 
            async_psycopg_copy, async_psycopg_executemany,
        )
```

File: _python/pgbench_python.py [hunk 6]
```diff
@@ -425,11 +489,21 @@ def die(msg):
...
+        connector, executor, copy_executor, batch_executor = \
+            psycopg2_connect, psycopg2_execute, psycopg2_copy, psycopg2_executemany
         is_async = False
         arg_format = 'python'
+    elif args.driver == 'psycopg3':
+        connector, executor, copy_executor, batch_executor = \
+            psycopg_connect, psycopg_execute, psycopg_copy, psycopg_executemany
+        is_async = False
+        arg_format = 'python'
+    elif args.driver == 'psycopg3[async]':
```
Comment (elprans) [inline _python/pgbench_python.py]:
```suggestion
    elif args.driver == 'psycopg3-async':
```

**Review by elprans — APPROVED** [review]
Looks good! Some minor formatting nits.

**Pull request merged**